package Part1.Exception

object Main {

}
